config.template

```diff
@@ -105,6 +127,7 @@ regionIndicesToPlot = [6]
 N_movavg = 12
 
 [nino34_timeseries]
+
 ## compare to output from another model run?
```

Comment:
I found is was hard to see the section names if they were crushed up against the next option but I could get behind having a comment about the whole section at the top instead of a space.